warehouse/utils/utils.go

```diff
@@ -934,3 +948,27 @@ func GetLoadFilePrefix(timeWindow time.Time, warehouse WarehouseT) (timeWindowFo
 	}
 	return timeWindowFormat
 }
+
+func GetRequestWithTimeout(ctx context.Context, url string, timeout time.Duration) ([]byte, int, error) {
```

Comment:
Why we need to add responseCode to the function signature ? Wouldn't []byte and error be enough ? Wdyt ?

Reply:
Done.